documentation/API_documentation/number-verification-API-Readiness-Checklist.md

```diff
@@ -1,21 +1,22 @@
-|  1 | API definition                               |   M   |         M         |    M    |    M   |   Y   | [link](/code/API_definitions/number-verification.yaml) |
-|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |   Y   |  [r2.3](https://github.com/camaraproject/Commonalities/releases/tag/r2.3)   |
-|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |   Y   |  [r2.3](https://github.com/camaraproject/IdentityAndConsentManagement/releases/tag/r2.3)   |
+|  1 | API definition                               |   M   |  M         |    M    |    M   |   Y   | [link](/code/API_definitions/number-verification.yaml) |
+|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |   Y   |  [r3.2](https://github.com/camaraproject/Commonalities/releases/tag/r3.2)   |
+|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |   Y   |  [r3.2](https://github.com/camaraproject/IdentityAndConsentManagement/releases/tag/r3.2)   |
 |  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |   Y   |      |
 |  5 | API documentation                            |   M   |         M         |    M    |    M   |   Y   | Embedded documentation into API spec - [link](/code/API_definitions/number-verification.yaml)  |
 |  6 | User stories                                 |   O   |         O         |    O    |    M   |   Y   | [get](/documentation/API_documentation/NumberVerification_device_phone_number_User_Story.md) [verify](/documentation/API_documentation/NumberVerification_verify_User_Story.md) |
-|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |   Y   | [get](code/Test_Definitions/number-verification-device-phone-number-share.feature) [verify](/code/Test_Definitions/number-verification-verify.feature) |
-|  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |   Y   | [get](code/Test_Definitions/number-verification-device-phone-number-share.feature) [verify](code/Test_Definitions/number-verification-verify.feature) |
+|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |   Y   | [get](code/Test_Definitions/number-verification-phoneNumberShare.feature) [verify](/code/Test_Definitions/number-verification-phoneNumberVerify.feature) |
+|  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |   Y   | [get](code/Test_Definitions/number-verification-phoneNumberShare.feature) [verify](code/Test_Definitions/number-verification-phoneNumberVerify.feature) |
 |  9 | Test result statement                        |   O   |         O         |    O    |    M   |   N   | see [issue #166](https://github.com/camaraproject/NumberVerification/issues/166) |

```

Comment:
the issues was for the previous version. Not needed for rc version, but at M4 release:

```suggestion
|  9 | Test result statement                        |   O   |         O         |    O    |    M   |   N   |     |
```